hail/python/setup-hailtop.py

```diff
@@ -1,11 +1,30 @@
+#!/usr/bin/env python
+
 from setuptools import setup, find_packages
 
 setup(
-    name = 'hailtop',
-    version = '0.0.1',
-    url = 'https://github.com/hail-is/hail.git',
-    author = 'Hail Team',
-    author_email = 'hail@broadinstitute.org',
-    description = 'Toplevel Hail module',
-    packages = find_packages()
+    name='hailtop',
+    version="0.0.1",
+    author="Hail Team",
+    author_email="hail-team@broadinstitute.org",
```

Comment:
This should be hail@broad...

Reply:
The `hail` PyPI package uses `hail-team@`. I changed them both to `hail@`.